src/config/Config.zig

```diff
@@ -5762,6 +5762,7 @@ pub const MacTitlebarProxyIcon = enum {
 pub const MacHidden = enum {
     never,
     always,
+    @"quick-terminal",
```

Comment:
Oh we need to add docs for this to the config key in this file.

Reply:
I added a comment for this new case, but maybe all the related paragraphs above need to be updated as well?